aws/sdk/integration-tests/s3/tests/express.rs

```diff
@@ -27,28 +27,6 @@ where
     aws_sdk_s3::Client::from_conf(update_builder(config).build())
 }
 
-// TODO(S3Express): Convert this test to the S3 express section in canary
-#[tokio::test]
```

Comment:
Is there any downside to also having this as an integration test?

Reply:
A connection recording test for `list_objects_v2` against an express bucket also exists in another test `mixed_auths`, so it's sort of already duplicated. Moving it to canary helped de-duplicate the test.